process.py

```diff
@@ -150,19 +150,19 @@ def import_accidents(provider_code, accidents, streets, roads):
             continue
         lng, lat = coordinates_converter.convert(accident[field_names.x_coordinate], accident[field_names.y_coordinate])
 
-        marker = Marker(
-            id=int("{0}{1}".format(provider_code, accident[field_names.id])),
-            title="Accident",
-            description=json.dumps(load_extra_data(accident, streets, roads), encoding=models.db_encoding),
-            address=get_address(accident, streets),
-            latitude=lat,
-            longitude=lng,
-            type=Marker.MARKER_TYPE_ACCIDENT,
-            subtype=int(accident[field_names.accident_type]),
-            severity=int(accident[field_names.accident_severity]),
-            created=parse_date(accident),
-            locationAccuracy=int(accident[field_names.igun]),
-        )
+        marker = {
```

Comment:
Yes, it's for SQLAlchemy Core. The *db.session.*execute method, takes the values for the insert command as a dictionary.